test/test_neuro_branch.py

```diff
@@ -161,6 +161,7 @@ def test_test_transform_maskatlas_error(self):
         # set the config so that caching works
         nb.set_params(**{'SmoothImages__fwhm': 10, 'ResampleImages__voxel_size': 5})
 
+        # todo: why does that fail?
```

Comment:
test_transform is currently build for saving 1,..,n images and requires .nii-file as output. BrainAtlas output is no nii-type.

Reply:
+1 thanks.